\subsection{16S rDNA Polymerase Chain Reaction\label{subsec-met-16s-pcr}}
For every reaction, a reaction volume of \SIul{20} was used. All components and the master mix were kept on ice until usage. The master mix was prepared by adding ultra-pure water first. Then, \SIul{4.0} high fidelity buffer (B0518 S, NEB), \SIul{0.4} \SIpM{100} forward primer 27f, \SIul{0.4} \SIpM{100} reverse primer 1492r or 1525r (all primers: TFS), \SIul{0.4} \SImM{10} dNTP mix, \SIul{0.2} high-fidelity polymerase (Phusion DNA Polymerase, NEB) and \SIul{1.0} sample.

Reaction tubes were placed in a PCR cycler (labcycler Gradient, SBE or MJMini, BRL), the lid closed and the program started. It consisted of the following steps:
\begin{itemize}
	\item \SIdC{95} for \SImin{10}
	\item Repeat 35 times:
	\begin{itemize}
		\item \SIdC{95} for \SIs{60}
		\item \SIdC{54} for \SIs{60}
		\item \SIdC{72} for \SIs{90}
	\end{itemize}
	\item \SIdC{72} for \SImin{10}
\end{itemize}
The lid temperature was \SIdC{105} and ultra-pure water was used as the default negative control. If available, a positive control was employed as well; usually genomic DNA which had already worked in a previous reaction.

\paragraph{Sequencing}
PCR products were cleaned up as described in \vref{subsec-gel-ex} and sequenced by GATC Biotech AG, Konstanz.

