\section{Strain Selection\label{sec-lch-eps-strain-selection}}
A set of seven strains was selected based not only on data presented within this work, but also taking into consideration data produced by other members of the working group and the expected scientific and industrial potential. The seven strains selected were:
\begin{itemize}
	\item \xyli{F6}: \mo{Paenibacillus}, also \epsj{B5}, screened on \acet{}
	\item \xyli{G11}: \mo{Pseudomonas}, also \epsi{G6}, screened on \fur{}, \hmf{}, \van{}, \acet{} and \fora{}
	\item \xyli{H10}: \mo{Sphingomonas}, also \epsj{A11}, screened on \van{} and \laev{}
	\item \xylj{A6}: \mo{Agrobacterium}, also \epsi{G12}, screened on \fur{}, \hmf{}, \van{}, \acet{} and \laev{}
	\item \xylj{B8}: \mo{Paenibacillus}, also \epsj{H7}, screened on \fur{}, \hmf{}, \acet{}, \fora{} and \laev{}
	\item \xylj{C4}: \mo{Curtobacterium}, also \epsi{B1}, screened on \fur{}, \hmf{}, \van{}, \acet{} and \laev{}
	\item \xylj{C11}: \mo{Rahnella}, also \epsi{B5}, screened on \van{}
\end{itemize}

%Datensammlung zum Stamm EPS2.H7:
%ursprüngliche Auswahl s. SK4, S. 200: EPS1.B1/Xyl2.C4/ISp.C4/ISp.D8/ISp.G11/ISr.F8/ISr.G12 (Curtobacterium), EPS1.B5/Xyl2.C11/ISp.E12 (Rahnella), EPS1.G6/Xyl1.G11/ISp.E3/ISp.B7/ISp.E10/ISr.A3/ISr.G6 (Pseudomonas), EPS1.G12/Xyl2.A6/ISp.G3/ISp.E7/ISp.C11/ISr.F6/ISr.B12 (Agrobacterium), EPS2.A11/Xyl1.H10/ISp.A11/ISr.E11 (Sphingomonas), EPS2.B5/Xyl1.F6/ISr.E6 (Paenibacillus), EPS2.H7/Xyl2.B8/ISp.B4/ISp.C8/ISr.G3/ISr.E8/ISr.F12 (Xanthomonas)
%

\nomenclature[latabbr_n.t.]{n.t.}{not tested}
\subsection{Strain Data Overview}
\begin{table}
	\centering
	\setlength{\tabcolsep}{5pt}
	\sisetup{
		table-number-alignment = center,
		table-text-alignment = center,
		table-figures-integer = 3,
		table-figures-decimal = 0,
		table-format = 3.0
	}
	\caption[\EPS{} Production in Inhibitor Presence of Selected Strains]{Summary of the cumulative aldose monomer concentrations of the \eps{}s of the selection of seven strains for more detailed analysis. Data are taken from \vref{tbl-inh-hcs-monomers}. Abbreviations: Fur.: \fur{}; HMF: \hmf{}; Van.: \van{}; Acet.: \acet{}; Form.: \fora{}; Laev.: \laev{}; n.t.: not tested.\label{tbl-lch-pf-strains-eps-summary}}
	\begin{tabular}{l*{6}{S}}
		\toprule
		 & \multicolumn{6}{c}{Aldose monomer concentration in \si{\milli\gram\per\litre}} \\
		\multirow{-2}*{Strain} & {Fur.} & {HMF} & {Van.} & {Acet.} & {Form.} & {Laev.} \\
		\hline
		{\xyli{F6}} & {n.t.} & {n.t.} & {n.t.} & 33 & {n.t.} & {n.t.} \\
		{\xyli{G11}} & 33 & 41 & 38 & 54 & 28 & {n.t.} \\
		{\xyli{H10}} & {n.t.} & {n.t.} & 9 & {n.t.} & {n.t.} & 91 \\
		{\xylj{A6}} & 219 & 924 & 33 & 680 & {n.t.} & 107 \\
		{\xylj{B8}} & 135 & 143 & {n.t.} & 139 & 216 & 131 \\
		{\xylj{C4}} & 187 & 273 & 217 & 350 & {n.t.} & 110 \\
		{\xylj{C11}} & {n.t.} & {n.t.} & 384 & {n.t.} & {n.t.} & {n.t.} \\
		\bottomrule
	\end{tabular}
\end{table}
All strains grew in the presence of \xyl{}, the three strains on plate Xyl1---\xyli{F6}, \xyli{G11}, \xyli{H10}---were part of the high-content screening on \xyl{} and they produced \SIlist{60;105;43}{\milli\gram\eps\per\litre}, respectively. All of the selected strains were part of at least one of the top 27/28 in the growth screening in the presence of inhibitors. The cumulative aldose monomer concentrations of all strains are summarized in \vref{tbl-lch-pf-strains-eps-summary}.

%Vorkommen in Screenings: xyl-hts: alle; xyl-hcs: nur Xyl1 (Xyl1.G11, Xyl1.H10, Xyl1.F6); inh-hts: alle
% inh-hcs: Xyl2.C4 (alle Ald., acet., laev.); Xyl1.G11 (alle Ald., form., acet.); Xyl2.A6 (alle Ald., acet., laev.); Xyl1.H10 (van., laev.); Xyl1.F6 (acet.); Xyl2.B8 (Fur., HMF, alle Säuren); Xyl2.C11 (van.); Stämme mit 1 g/l: keiner; Monomersummen (mg/l): Xyl2.C4: 160/107/233/353/119; Xyl1.G11: 46/47/51/29/59; Xyl2.A6: 43/237/942/118/364; Xyl1.H10: 39/104; Xyl1.F6: 28; Xyl2.B8: 153/172/165/142/227; Xyl2.C11: 361.

\subsection{Further Analyses\label{subsec-lch-eps-strain-selection-further}}
\nomenclature[latabbr_n.d.]{n.d.}{not detected}
\nomenclature[chem_GalN]{GalN}{\textsc{d}-galactosamine}
\nomenclature[chem_GlcNAc]{GlcNAc}{\textit{N}-acetyl-\textsc{d}-glucosamine}
\begin{table}
	\centering
	\setlength{\tabcolsep}{5pt}
	\caption[\EPS{} \AMC{}s of the Four Remaining Strains]{\EPS{} \amc{}s of the four remaining strains. The four strains \xyli{F6}, \xyli{H10}, \xylj{B8} and \xylj{C11} were incubated in \SIml{20} SM1 P100 for \SId{4} at \SIrpm{150}. The \eps{}s were recovered using centrifugation and isopropanol precipitation. After drying, solutions with \SIrange{1.0}{10}{\gram\eps\per\litre} were prepared and the \amc{} determined. The recovery was calculated by dividing the cumulative concentration of all aldose monomers by the \eps{} concentration used. For \xylj{B8}, filamentous precipitate floating on top and a pellet were found. The filamentous precipitate is indicated by $^{\text{top}}$. The data are visualized in \vref{fig-lch-pf-strain-selection-further-eps-composition}. Abbreviations: Gal:~\gal{}; GalN:~\textsc{d}-galactosamine; Glc:~\glc{}; GlcN:~\textsc{d}-glucosamine; GlcNAc:~\textit{N}-acetyl-\textsc{d}-glucosamine; GlcUA:~\textsc{d}-glucuronic acid; Man:~\textsc{d}-mannose; Rha:~\textsc{l}-rhamnose; Rib:~\textsc{d}-ribose; Sum:~cumulative concentration of all aldose monomers; n.d.: not detected.\label{tbl-lch-pf-strain-selection-further-eps-composition}}
	\begin{tabular}{l*{10}{r}S[table-format=2.0]}
		\toprule
		 & \multicolumn{10}{c}{Aldose monomer concentration in \si{\milli\gram\per\litre}} & \\
		\multirow{-2}*{Strain} & {Gal} & {GalN} & {Glc} & {GlcN} & {GlcNAc} & 
		{GlcUA} & {Man} & {Rha} & {Rib} & {Sum} & {\multirow{-2}*{Recovery}} \\
		\hline
		\TablesafeInputIfFileExists{data/lch-eps/strain/scm0-results.tex}{}{\fxfatal{File not found: data/lch-eps/strain/scm0-results.tex}}
		\bottomrule
	\end{tabular}
\end{table}

\begin{figure}
	\begin{center}
		\includegraphics[width=\textwidth]{fig/strain_scm0_600dpi.png}
		\caption[\EPS{} \AMC{}s of the Four Remaining Strains]{\EPS{} \amc{}s of the four remaining strains. The four strains \xyli{F6}, \xyli{H10}, \xylj{B8} and \xylj{C11} were incubated in \SIml{20} SM1 P100 for \SId{4} at \SIrpm{150}. The \eps{}s were recovered using centrifugation and isopropanol precipitation. After drying, solutions with \SIrange{1.0}{10}{\gram\eps\per\litre} were prepared and the \amc{} determined.  For \xylj{B8}, filamentous precipitate floating on top and a pellet were found. The filamentous precipitate is indicated by \enquote{top}. The concentrations of the filamentous precipitate and the pellet of \xylj{B8} differed by a factor of 10. Therefore, some monomer concentrations were below the detection limit in the \enquote{top} sample. The data for this figure are given in \vref{tbl-lch-pf-strain-selection-further-eps-composition}.\label{fig-lch-pf-strain-selection-further-eps-composition}}
	\end{center}
\end{figure}

In order to familiarize with each strain, they were grown on agar plates of AMA, LB, SM1 P30S and ST1, incubated for two to four days at \SIdC{30} and colony colour and morphology were noted down. Also, the strains were incubated in \SIml{10} LB and SM1 P30S in \SIml{50} baffled Erlenmeyer flasks for three days, the broth diluted to \SIml{50} and centrifuged for \SImin{15} at \SIG{3000} and \SIdC{20}. The supernatant was precipitated in a 2:1 ratio using isopropanol or ethanol (see \vref{subsec-met-precipitation}) without any visible precipitation.

Genomic DNA of all strains was isolated in order to run a 16S PCR on it and as a starting material for genome sequencing. Due to issues with the 16S PCR and gel purification, the 16S rDNA sequence was only available after the strain had been selected. Although the genomes of the strains \xyli{G11}, \xylj{A6} and \xylj{C4} were extracted, they were not considered for further analysis due to purity concerns after incubation on agar plates.
%SK5, ab S. 4: Wachstum auf Agar: AMA, LB, SM1 P30S, ST1 (2 d bis 4 d); Wachstum auf Medium: LB, SM1 P30S (2015-01-27, 16:30 Uhr bis 2015-01-30, 14:50 Uhr = ~3 d), Testfällungen; Genomextraktionen: alle
% S. 20: EPS1.B1, EPS1.G6, EPS1.G12 verwerfen, da nicht sauber
% ff.: 16S-PCR: nur von EPS2.H7, da Negativkontrollen positiv oder Aufreinigung nie geklappt bei allen, dann am Ende nur noch der tatsächlich genutzte Stamm interessant

The remaining strains were incubated in \SIml{20} SM1 P100 in \SIml{100} baffled Erlenmeyer flasks at \SIdC{30} and \SIrpm{150} for four days. The broths were diluted to \SIml{50} and centrifuged for \SImin{30} at \SIG{4000} and \SIdC{30}. The supernatant was still turbid---only slightly for \xyli{H10} and \xylj{B8}---in all cases, but pellets had formed nonetheless. The solutions were precipitated with isopropanol at a 2:1 ratio (see \vref{subsec-met-precipitation}). The only supernatant giving directly visible filamentous precipitates was \xylj{B8}'s. There was noticeable precipitate in all samples. The filamentous precipitates of \xylj{B8} floating on the top and the pellet at the bottom were analysed separately.
%S. 36, 46-48: alle außer EPS1.G6, EPS1.G12: 2015-02-16, 11:50 Uhr, SM1 P100; bis: 2015-02-20, 11:35 Uhr (= 4 d); nur bei 2.H7 fädiges Produkt!

After the precipitate had settled for a month, the supernatant was removed carefully, the pellets air-dried for \SIh{2} and then dried in a vacuum drying oven at \SIdC{45} for \SIh{20}. The dried pellets were weighed and used to calculate the approximate \eps{} concentrations in the broths:

\begin{itemize}
	\item \xyli{F6}: \SIgpl{7.3}
	\item \xyli{H10}: \SIgpl{0.2}
	\item \xylj{B8}: \SIgpl{2.0}
	\item \xylj{C11}: \SIgpl{8.4}
\end{itemize}

These pellets were dissolved in ultra-pure water to yield final concentrations of \SIgpl{1.0} for \xyli{F6}, \xylj{B8} (top) and \xylj{C11}, \SIgpl{4.0} for \xyli{H10} and \SIgpl{10} for \xylj{B8}. The \eps{} aldose compositions are given in \vref{tbl-lch-pf-strain-selection-further-eps-composition} and are shown in \vref{fig-lch-pf-strain-selection-further-eps-composition}.
%S. 80: Trockenmassen, Einstellung der Lösungen; Fehler in Auswertung: 2.H7 hat ebenfalls KOnzentration von 10 g/l; %S. 81: Fehler in Auswertung: 2.H7oben hat KOnzentration von 1 g/l --> passt auch besser zu Daten --> Zusammensetzungen und Recovery aus 2.H7 und 2.H7oben gleich!!!; S. 90: Zusammenfassung/Übersicht; Rücklösbarkeit

\subsection{Selection of \strain{}\label{subsec-lch-eps-strain-selection-winner}}
The strain \xylj{B8}---tentatively named \enquote{\strain{}} after its location on the EPS plates, EPS2.H7---was chosen as the \eps{} production strain for the subsequent parallel fermentation. The strain showed robust growth on different media and \eps{} production in the presence of different inhibitors. The robustness of the \eps{} production was derived from \vref{tbl-lch-pf-strains-eps-summary}. The aldose monomer concentration of \xylj{B8} in the presence of each of the five inhibitors tested lies within a range of \SIrange{131}{216}{\milli\gram\per\litre}, while every other strain tested with five inhibitors either produced low amounts of \eps{} only (\xyli{F6}) or showed a greater variance in the \eps{} concentration (\xylj{A6}, \xylj{C4}). The product could be purified with relative ease and the product concentration was sufficient; the desired minimal concentration was \SIgpl{1.0}. The \eps{} \amc{} showed a total of eight different monomers dominated by \glc{} and \man{}. The presence of acidic and basic monomers could contribute to unusual rheological properties making the resulting polymer a candidate for possible future industrial uses. Therefore, the \eps{} appeared to have a high potential and \strain{} was selected for further studies.

\nomenclature[mo_P. cineris]{\mo{P. cineris}}{\mo{Paenibacillus cineris}}
\nomenclature[mo_P. favisporus]{\mo{P. favisporus}}{\mo{Paenibacillus favisporus}}
\nomenclature[mo_P. azoreducens]{\mo{P. azoreducens}}{\mo{Paenibacillus azoreducens}}
\subsubsection{16S rDNA Sequence}
\begin{table}
	\centering
	\setlength{\tabcolsep}{5pt}
	\caption[16S rDNA BLAST Results for \strain{}]{16S rDNA BLAST results for \strain{}. The 16S rDNA sequence of \strain{} was used in a MegaBLAST query and the first ten results are given in this table. E values were \num{0.0} in all cases.\label{tbl-lch-eps-strain-blast-top-ten}}
	\begin{tabular}{lrrr}
		\toprule
		{Description} & {Identifier} & {Score in bits} & {Reference} \\
		\hline
		{\mo{Paenibacillus} sp.~A25 16S rRNA, partial} & {KF479541.1} & {2634} & {n.a.} \\
		{\mo{Paenibacillus} sp.~SSG-1 16S rRNA, partial} & {KF750627.1} & {2623} & {\cite{Song2014}} \\
		{\mo{Paenibacillus} sp.~C82 16S rRNA gene, partial} & {JX011004.1} & {2619} & {n.a.} \\
		{\mo{Paenibacillus cineris} 16S rRNA gene, partial} & {LN890143.1} & {2617} & {n.a.} \\
		{\mo{Paenibacillus} sp.~B19 16S rRNA gene, partial} & {KF479580.1} & {2617} & {n.a.} \\
		{\mo{Paenibacillus cineris}~JN237 16S rRNA gene, partial} & {KF150476.1} & {2617} & {n.a.} \\
		{\mo{Paenibacillus favisporus}~T2 16S rRNA gene, partial} & {JN867753.1} & {2612} & {n.a.} \\
		{\mo{Paenibacillus} sp.~3492BRRJ 16S rRNA gene, partial} & {JF309261.1} & {2612} & {n.a.} \\
		{\mo{Paenibacillus} sp.~07-G-dH 16S rRNA gene, partial} & {HM776458.1} & {2610} & {\cite{Winyasuk2012}} \\
		{\mo{Paenibacillus} sp.~FJAT-21993 16S rRNA gene, partial} & {KP728976.1} & {2608} & {n.a.} \\
		\bottomrule
	\end{tabular}
\end{table}
Genomic DNA was extracted and used as template for a 16S rDNA amplification. The complete sequence is given in \vref{fig-lch-pf-strain-16s} and was used in a BLAST search using MegaBLAST \cite{Zhang2004,Morgulis2008} in the database \enquote{nt} of NCBI. The ten highest scoring hits are given in \vref{tbl-lch-eps-strain-blast-top-ten}, E values were \num{0} for at least the 100 highest scoring sequences. Among these, the dominating genera were \mo{Paenibacillus} (90) and \mo{Bacillus} (6), the most frequently found species were \mo{P. cineris} (11), \mo{P. favisporus} (11) and \mo{P. azoreducens} (6). % Database Posted date:  Jan 19, 2016  1:46 PM
